v3/plugins/ms-kubernetes-tools/vscode-kubernetes-tools/1.0.4/meta.yaml

```diff
@@ -1,19 +1,19 @@
 apiVersion: v2
 publisher: ms-kubernetes-tools
 name: vscode-kubernetes-tools
-version: latest
+version: 1.0.4
 type: VS Code extension
 displayName: Kubernetes
 title: Kubernetes Tools
 description: Develop, deploy and debug Kubernetes applications
 icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
 repository: https://github.com/Azure/vscode-kubernetes-tools
 category: Other
-firstPublicationDate: '2019-05-15'
+firstPublicationDate: "2019-10-16"
 spec:
   containers:
-  - image: "docker.io/eclipse/che-remote-plugin-kubernetes-tooling-1.0.0:7.3.0"
-    name: vscode-kubernetes-tools
-    memoryLimit: "1G"
+    - image: "docker.io/eclipse/che-remote-plugin-kubernetes-tooling-1.0.4:next"
```

Comment:
Should be tagged `7.3.1`? This is the "latest" version.

Reply:
@amisevsk good catch, I assume if we would like to keep `pizzafactory` plugins in `7.3.1` the container runtime image should be also updated there

Reply:
:+1: @monaka any issues with this?